platform/iphone/app_delegate.mm

```diff
@@ -604,7 +595,14 @@ - (void)applicationDidFinishLaunching:(UIApplication *)application {
 	glGetRenderbufferParameterivOES(GL_RENDERBUFFER_OES,
 			GL_RENDERBUFFER_HEIGHT_OES, &backingHeight);
 
-	int err = iphone_main(backingWidth, backingHeight, gargc, gargv);
+	NSArray *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory,
+			NSUserDomainMask, YES);
+	NSString *documentsDirectory = [paths objectAtIndex:0];
+	// NSString *documentsDirectory = [[[NSFileManager defaultManager]
```

Comment:
Can we kill these comments?

Reply:
Done, but there is plenty of other commented chunks of code in app_delegate.mm, perhaps it should be cleaned up separately. @BastiaanOlij are they necessary as some reference or can they be removed freely?

Reply:
I think for now the idea is to kill commented out code as we encounter it, I don't think we're currently looking to just kill all of it with separate commits.